resource-managers/kubernetes/core/pom.xml

```diff
@@ -29,7 +29,7 @@
   <name>Spark Project Kubernetes</name>
   <properties>
     <sbt.project.name>kubernetes</sbt.project.name>
-    <kubernetes.client.version>1.4.34</kubernetes.client.version>
+    <kubernetes.client.version>2.0.3</kubernetes.client.version>
```

Comment:
None as far as I've checked. It looks like they just chose a sudden version jump a few days ago: https://github.com/fabric8io/kubernetes-client/releases?after=kubernetes-client-2.0.0.fuse-000002